### PR TITLE
fix: update ModelScopeEstimator image

### PR DESCRIPTION
**Pull request by YuZ1225**
Update ModelScopeEstimator image. Support latest official ModelScope image.

### CODE REVIEW

File: pai/modelscope/estimator.py [hunk 1]
```diff
@@ -196,6 +196,8 @@ def __init__(
             session=session,
             **kwargs,
         )
+        # Check image_uri and modelscope_version
+        self.training_image_uri()
```
Comment (pitt-liang) [inline pai/modelscope/estimator.py:200]: check if the parameters is valid in `self._validate_image_uri`?

File: pai/modelscope/estimator.py [hunk 2]
```diff
@@ -219,10 +221,9 @@ def training_image_uri(self) -> str:
 
         labels = [
             ImageLabel.OFFICIAL_LABEL,
-            ImageLabel.DLC_LABEL,
-            ImageLabel.PROVIDER_COMMUNITY_LABEL,
+            ImageLabel.DSW_LABEL,
             ImageLabel.DEVICE_TYPE_GPU,
-            ImageLabel.framework_version("PyTorch", "*"),
+            ImageLabel.framework_version("ModelScope", "*"),
```
Comment (pitt-liang) [inline pai/modelscope/estimator.py:226]: Dulicate Framework filter label? 

```
        if self.modelscope_version == "latest":
            latest_version = self._get_latest_ms_version_for_training()
            labels.append(ImageLabel.framework_version("ModelScope", latest_version))
        else:
            labels.append(
                ImageLabel.framework_version("ModelScope", self.modelscope_version)
            )


```

**Review by pitt-liang — CHANGES_REQUESTED** [review]

**Review by pitt-liang — APPROVED** [review]
